src/components/HashLink/PHashLink.vue

```diff
@@ -40,6 +40,7 @@
   cursor-pointer
   relative
   transition-all
+  flex
```

Comment:
What's the reason for this change?

Reply:
So you can put a `p-heading` inside of a hashlink. Otherwise the hash and the text are on separate lines. This going to cause issues? I can revert. Only reason I added this was to use the p-heading component on the demo page

Reply:
Ahh - this may need to be updated in a few places then, the change breaks this section of the markdown renderer and probably elsewhere that I haven't thought to check:
<img width="1105" alt="Screen Shot 2023-03-08 at 12 15 20 PM" src="https://user-images.githubusercontent.com/27291717/223783333-a3c4fb3d-009b-416a-83a6-7563c09d742e.png">

Reply:
ahh good catch. Pushed a fix for that in the markdown parser. I cannot find any other uses of PHashLink outside of the demo in any of our projects.